### PR TITLE
Feat: transaction details in tx modal

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves
Adds transaction details when signing/executing an existing tx.

Screenshot 2022-08-19 at 11 48 30

### CODE REVIEW

File: src/components/transactions/TxSummary/index.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ const TxSummary = ({ item, isGrouped }: TxSummaryProps): ReactElement => {
       )}
 
       {wallet && !isWrongChain && isQueue && (
-        <Box gridArea="actions">
+        <Box gridArea="actions" display="flex" justifyContent="center" gap={1}>
```
Comment (katspaugh) [inline src/components/transactions/TxSummary/index.tsx:90]: Screenshot 2022-08-19 at 11 53 24

File: src/components/tx/DecodedTx/index.tsx [hunk 2]
```diff
@@ -1,44 +1,64 @@
...
+
           {decodedData ? (
             <MethodDetails data={decodedData} />
-          ) : error ? (
-            <ErrorMessage error={error}>Failed decoding transaction data</ErrorMessage>
+          ) : decodedDataError ? (
+            <ErrorMessage error={decodedDataError}>Failed decoding transaction data</ErrorMessage>
           ) : (
-            <Skeleton />
+            decodedDataLoading && <Skeleton />
```
Comment (usame-algan) [inline src/components/tx/DecodedTx/index.tsx:61]: Should we display this part above the details since the method is more general than the tx details?

Comment (katspaugh) [reply src/components/tx/DecodedTx/index.tsx:61]: I put it below because the last item in tx details is "raw data", so it kinda continues it nicely.

Comment (katspaugh) [reply src/components/tx/DecodedTx/index.tsx:61]: Also I don't see it as more general. All txns have a safeTxHash, but not all have any tx data (e.g. native transfers don't).

**Review by usame-algan — APPROVED** [review]
🚀

**Pull request merged**